test/test_logging.py

```diff
@@ -237,3 +245,29 @@ def test_set_log_level_with_bad_level_uses_info(set_console_ip, debug_threshold,
             {'level': 'CRITICAL', 'text': 'hello critical'},
             {'level': 'ERROR', 'text': 'hello exception'}],
                                 not_expected_lines=[{'level': 'DEBUG', 'text': 'hello debug'}])
+
+def test_syslog_format(set_console_ip, debug_threshold, tmpdir):
```

Comment:
We may need to rethink this test. Hijacking one formatter to test the other is a bit too hacky.